docs/tutorials/02_vqe_advanced_options.ipynb

```diff
@@ -0,0 +1,549 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Callback"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "Callback methods can be used to monitor optimization progress as the algorithm runs and converges to the minimum. The callback is invoked for each functional evaluation by the optimizer and provides the current optimizer value, evaluation count, current optimizer parameters etc. Note that, depending on the specific optimizer this may not be each iteration (step) of the optimizer, so for example if the optimizer is calling the cost function to compute a finite difference based gradient this will be visible via the callback.\n",
+    "\n",
+    "This section demonstrates how to leverage callbacks in `VQE` to plot the convergence path to the ground state energy with a selected set of optimizers."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, you need a qubit operator for VQE. For this example, you can use the same operator as used in the algorithms introduction, which was originally computed by [Qiskit Nature](https://qiskit.org/ecosystem/nature/) for an H2 molecule."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit.quantum_info import SparsePauliOp\n",
+    "\n",
+    "H2_op = SparsePauliOp.from_list(\n",
+    "    [\n",
+    "        (\"II\", -1.052373245772859),\n",
+    "        (\"IZ\", 0.39793742484318045),\n",
+    "        (\"ZI\", -0.39793742484318045),\n",
+    "        (\"ZZ\", -0.01128010425623538),\n",
+    "        (\"XX\", 0.18093119978423156),\n",
+    "    ]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The next step is to instantiate the `Estimator` of choice for the evaluation of expectation values within `VQE`. For simplicity, you can select the [qiskit.primitives.Estimator](https://qiskit.org/documentation/stubs/qiskit.primitives.Estimator.html#qiskit.primitives.Estimator) shipped with the default [Qiskit](https://qiskit.org/documentation/apidoc/terra.html) installation."
```

Comment:
> shipped with the default [Qiskit](https://qiskit.org/documentation/apidoc/terra.html) installation."

Should we change this to qiskit,.org or perhaps just point at the documentation and drop the apidoc/terra.html (I am not sure how that will change) there is also an install page since this seems about install - or we could just reword it a bit to sat these come as part of Qiskit (with whatever Qiskit there then links to)